packages/frontend/src/layouts/MainView.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Route } from 'react-router-dom';
+import HomePage from '../pages/HomePage';
+import Movies from '../pages/Movies';
+import Series from '../pages/Series';
+
+const MainView: React.FC = () => {
+    return (
+        <>
+            <Route path="/" exact component={HomePage} />
+            <Route path="/filmy" component={Movies} />
```

Comment:
A ścieki mozna utworzyć jako enumy w osobnym pliku, ale to juz jak kto woli :)

Reply:
Url do enum bo bedziemy go potrzebowac potem do przekierowan. Gdy url jest w enum to gdy zmienimy url z jakiegos powodu nie musimy podmieniac recznie url w wszystkich przyciskach ktore ww. url uzywaja

Reply:
Stworzyłam enum: 

```
enum NAVIGATION {
    HOMEPAGE = '/',
    MOVIES = '/filmy',
    SERIES = '/seriale'
}
```

Który importuje w Navigation.tsx oraz MainView.tsx